--- a/config.def.h	2020-12-16 14:22:00.389211087 +0100
+++ b/config.def.h 2020-12-16 14:05:55.379287051 +0100
@@ -34,7 +34,7 @@ static float chscale = 1.0;
  *
  * More advanced example: L" `'\"()[]{}"
  */
-wchar_t *worddelimiters = L" ";
+wchar_t *worddelimiters = L" `'\"()[]{}";
 
 /* selection timeouts (in milliseconds) */
 static unsigned int doubleclicktimeout = 300;
@@ -73,8 +73,8 @@ static unsigned int cursorthickness = 2;
  *    Bold affects lines thickness if boxdraw_bold is not 0. Italic is ignored.
  * 0: disable (render all U25XX glyphs normally from the font).
  */
-const int boxdraw = 0;
-const int boxdraw_bold = 0;
+const int boxdraw = 1;
+const int boxdraw_bold = 1;
 
 /* braille (U28XX):  1: render as adjacent "pixels",  0: use font */
 const int boxdraw_braille = 0;
@@ -86,7 +86,7 @@ const int boxdraw_braille = 0;
 static int bellvolume = 0;
 
 /* default TERM value */
-char *termname = "st-256color";
+char *termname = "xterm-256color";
 
 /*
  * spaces per tab
@@ -143,7 +143,7 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 258;
+unsigned int defaultfg = 15;
 unsigned int defaultbg = 259;
 unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
@@ -195,7 +196,7 @@ static uint forcemousemod = ShiftMask;
  * Xresources preferences to load at startup
  */
 ResourcePref resources[] = {
-		{ "font",         STRING,  &font },
+		{ "font",         STRING,  &font }, 
 		{ "color0",       STRING,  &colorname[0] },
 		{ "color1",       STRING,  &colorname[1] },
 		{ "color2",       STRING,  &colorname[2] },
@@ -225,6 +226,7 @@ ResourcePref resources[] = {
 		{ "borderpx",     INTEGER, &borderpx },
 		{ "cwscale",      FLOAT,   &cwscale },
 		{ "chscale",      FLOAT,   &chscale },
+		{ "alpha",        FLOAT,   &alpha },
 };
 
 /*
@@ -233,13 +235,9 @@ ResourcePref resources[] = {
  */
 static MouseShortcut mshortcuts[] = {
 	/* mask                 button   function        argument       release */
-	{ ShiftMask,            Button4, kscrollup,      {.i = 1} },
-	{ ShiftMask,            Button5, kscrolldown,    {.i = 1} },
 	{ XK_ANY_MOD,           Button2, selpaste,       {.i = 0},      1 },
-	{ ShiftMask,            Button4, ttysend,        {.s = "\033[5;2~"} },
-	{ XK_ANY_MOD,           Button4, ttysend,        {.s = "\031"} },
-	{ ShiftMask,            Button5, ttysend,        {.s = "\033[6;2~"} },
-	{ XK_ANY_MOD,           Button5, ttysend,        {.s = "\005"} },
+        { XK_ANY_MOD,           Button4, kscrollup,       {.i = 3} },
+        { XK_ANY_MOD,           Button5, kscrolldown,     {.i = 3} },
 };
 
 /* Internal keyboard shortcuts. */
@@ -452,17 +450,17 @@ static Key key[] = {
 	{ XK_F5,            XK_NO_MOD,      "\033[15~",      0,    0},
 	{ XK_F5, /* F17 */  ShiftMask,      "\033[15;2~",    0,    0},
 	{ XK_F5, /* F29 */  ControlMask,    "\033[15;5~",    0,    0},
-	{ XK_F5, /* F41 */  Mod4Mask,       "\033[15;6~",    0,    0},
+	{ XK_F5, /* F41 */  Mod4Mask,       "\033[15~",    0,    0},
 	{ XK_F5, /* F53 */  Mod1Mask,       "\033[15;3~",    0,    0},
 	{ XK_F6,            XK_NO_MOD,      "\033[17~",      0,    0},
 	{ XK_F6, /* F18 */  ShiftMask,      "\033[17;2~",    0,    0},
 	{ XK_F6, /* F30 */  ControlMask,    "\033[17;5~",    0,    0},
-	{ XK_F6, /* F42 */  Mod4Mask,       "\033[17;6~",    0,    0},
+	{ XK_F6, /* F42 */  Mod4Mask,       "\033[17~",    0,    0},
 	{ XK_F6, /* F54 */  Mod1Mask,       "\033[17;3~",    0,    0},
 	{ XK_F7,            XK_NO_MOD,      "\033[18~",      0,    0},
 	{ XK_F7, /* F19 */  ShiftMask,      "\033[18;2~",    0,    0},
 	{ XK_F7, /* F31 */  ControlMask,    "\033[18;5~",    0,    0},
-	{ XK_F7, /* F43 */  Mod4Mask,       "\033[18;6~",    0,    0},
+	{ XK_F7, /* F43 */  Mod4Mask,       "\033[18~",    0,    0},
 	{ XK_F7, /* F55 */  Mod1Mask,       "\033[18;3~",    0,    0},
 	{ XK_F8,            XK_NO_MOD,      "\033[19~",      0,    0},
 	{ XK_F8, /* F20 */  ShiftMask,      "\033[19;2~",    0,    0},
